lib/src/visitor/async_evaluate.dart

```diff
@@ -2884,22 +2925,24 @@ class _EvaluateVisitor
 
   /// Returns the [AstNode] whose span should be used for [expression].
   ///
-  /// If [expression] is a variable reference, [AstNode]'s span will be the span
-  /// where that variable was originally declared. Otherwise, this will just
-  /// return [expression].
+  /// If [expression] is a variable reference and [_sourceMap] is `true`,
```

Comment:
Isn't `_sourceMap` no longer relevant for this function? (at least until the /-as-division warnings are gone)